Xpirit-Vsts-Release-Terraform/terraform.ps1

```diff
@@ -127,6 +127,10 @@ function Initialize-Terraform
     $additionalArguments = Get-VstsInput -Name InitArguments
 
     $extraArguments = " -input=false -no-color"
+    
+    if ($additionalArguments.Trim() -like "validate*") {
```

Comment:
This is not the right place, I bet you didn't test it :)
This is for the init command and not the actual command.